app/src/main/java/org/akvo/flow/ui/view/QuestionGroupTab.java

```diff
@@ -201,15 +201,31 @@ private void updateGroupIterationHeaders() {
         }
     }
 
+    //TODO: this method should be refactored to find a better way to distinguish between
```

Comment:
Create a separate issue for this and then we can clear out this comment

Reply:
Done https://github.com/akvo/akvo-flow-mobile/issues/1557